src/app/modules/verification/__tests__/verification.service.spec.ts

```diff
@@ -1047,29 +1053,58 @@ describe('Verification service', () => {
           }),
         ],
       }
+      const recipient = MOCK_LOCAL_RECIPIENT
       const fieldIdOtherString = new ObjectId().toHexString()
 
       // Act
       const actual = await VerificationService.shouldGenerateMobileOtp(
         mockForm,
         fieldIdOtherString,
+        recipient,
       )
 
       // Assert
-      expect(actual._unsafeUnwrapErr()).toBeInstanceOf(OtpRequestError)
+      expect(actual._unsafeUnwrapErr()).toBeInstanceOf(MalformedParametersError)
     })
 
-    it('should return OtpRequestError if form_fields is empty', async () => {
+    it('should return MalformedParametersError if form_fields is empty', async () => {
```

Comment:
did these tests not fail :O

Reply:
ah the ci pipeline bugged out; i've cancelled them as they most definitely will fail haha

Reply:
uh oh